src/editor/components/graph/node.tsx

```diff
@@ -9,22 +25,44 @@ type NodeData = {
   params?: any,
   canAddTargetHandle?: boolean,
   dataFlowId?: string,
-  onAddedTargetHandle?: ( e: any ) => void
+  docsMdPath: string,
+  onAddedTargetHandle?: ( e: any ) => void,
 };
 
 export type FlooqNode = ReactFlowNode<NodeData>;
 
-export const Node: FC<FlooqNode> = ( { id, data, children } ) => {
+export const Node: FC<FlooqNode> = ( { id, data, type: nodeType, children } ) => {
   const reactFlowHook = useReactFlow()
 
+  const [areDocsOpen, setAreDocsOpen] = useState( false )
+  const [docsContent, setDocsContent] = useState<string>( '' )
+
+  if ( docsContent.length === 0 && nodeType ) {
+    const path = docsPaths[nodeType as NodeType]
+    if ( path )
+      fetch( `/api/flows/docs?${new URLSearchParams( { path } )}` )
+        .then( res => res.ok ? res : Promise.reject( res.statusText ) )
+        .then( res => res.text() )
+        .then( res => setDocsContent( res ) )
+        .catch( e => setDocsContent( `No documentation found for this Node.` ) )
+  }
+
   const deleteNode = (): any => {
     reactFlowHook.setNodes( reactFlowHook.getNodes().filter( n => n.id !== id ) )
     reactFlowHook.setEdges( reactFlowHook.getEdges().filter( e => e.target !== id && e.source !== id ) )
   }
 
   return (
-    <div className="flex bg-gray-100 dark:bg-gray-900">
-      {data.incomingHandles &&
+    <>
+      <Dialog
+        isOpen={ areDocsOpen }
+        onClose={ (): void => setAreDocsOpen( false ) }>
+        <Markdown className="dark:text-gray-100 pb-5 markdown" options={{ forceBlock: true }}>
+          {docsContent}
+        </Markdown>
+      </Dialog>
```

Comment:
I would add this in a separate component. I know it's not much but for this component I would like a storybook entry.